src/screens/Main.tsx

```diff
@@ -94,11 +94,13 @@ function Main() {
               {searchItems(index)?.component}
             </Grid>
           </Grid>
-          <Grid container>
-            <Grid item md={12}>
-              <Transactions />
+          {index !== 3 && (
```

Comment:
`index !== 3` is ugly, can you use a constant or at least extract to a variable, like so:
```
const isConnectToWalletScreen = index === CONNECT_WALLET_SCREEN;
```